common/pom.xml

```diff
@@ -63,12 +63,29 @@
       <groupId>org.msgpack</groupId>
       <artifactId>msgpack</artifactId>
       <version>0.6.6</version>
+      <exclusions>
```

Comment:
Exclusions should not be necessary.

Reply:
Removed exclusion for msgpack. The exclusion of javassist for reflections is necessary because before javassist version 3.13 the groupId was **javassist** after version 3.13 the groupId of javassist was changed to **org.javassist**. I also noticed ./logical/pom.xml declared the same msgpack and reflections dependencies as drill common. So I removed the redundant dependency declaration and also made msgpack and reflections managed dependencies.